### ISSUE TITLE
Landscape full screen bug

### ISSUE THREAD

**Issue by pedro1977uk** [TeamNewPipe/NewPipe#8727]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.23.1

### Steps to reproduce the bug

Set screen to fill.
Play live channel like sky news UK
Rotate to landscape.
Doesn't happen every time but only half the screen (top) is filled with the entire image. If I rotate back and forth, eventually it will work correctly. Problem reoccurs when starting stream again.

### Expected behavior

Should fill entire screen when rotating to landscape.

### Actual behavior

Only fills half / top half of the screen.  

### Screenshots/Screen recordings

![Screenshot_2022-08-02-23-57-31-373_org schabi newpipe](https://user-images.githubusercontent.com/68713442/182487506-2671a021-846d-43a9-8331-b7415caef40d.jpg)

### Affected Android/Custom ROM version

Android 12 miui 13

### Affected device model

Mi 10 5g, poco f3

**Comment by mhmdanas** [comment]
May be related to https://github.com/TeamNewPipe/NewPipe/issues/7262.

Are you able to reproduce the issue with [the current `master` APK](https://github.com/TeamNewPipe/NewPipe/suites/7667138854/artifacts/319623171)?

**Comment by Invisigon1** [comment]
This issue seems not to happen on Android 13 Beta 4. It did happen to me on Android 12 before.

**Comment by hammolo** [comment]
Is surely related to https://github.com/TeamNewPipe/NewPipe/issues/7262 (no idea why it was closed tbh). I have this issue since a year with two different phones by Samsung and Motorola. Both Android 12. Depending on the fill type you set, you get the top left angle, or the top 50% of the phone.

**Comment by mhmdanas** [comment]
> Is surely related to https://github.com/TeamNewPipe/NewPipe/issues/7262 (no idea why it was closed tbh).

Correction: _maybe_ related. The issue was closed because users tested #8170 and stated that it fixed the issue.

> I have this issue since a year with two different phones by Samsung and Motorola. Both Android 12. Depending on the fill type you set, you get the top left angle, or the top 50% of the phone.

Does that happen with the latest NewPipe version (0.23.2) too?

**Comment by pedro1977uk** [comment]
Yes it still happens on the latest version but maybe a little less often.
 
 
  On Thu, 18 Aug 2022 at 15:38, Mohammed ***@***.***> wrote:   



Is surely related to #7262 (no idea why it was closed tbh).


Correction: may be related. The issue was closed because users tested #8170 and stated that it fixed the issue.


I have this issue since a year with two different phones by Samsung and Motorola. Both Android 12. Depending on the fill type you set, you get the top left angle, or the top 50% of the phone.


Does that happen with the latest NewPipe version (0.23.2) too?

**Comment by hammolo** [comment]
> > Is surely related to #7262 (no idea why it was closed tbh).
> 
> Correction: _may be_ related. The issue was closed because users tested #8170 and stated that it fixed the issue.

I see.

> 
> > I have this issue since a year with two different phones by Samsung and Motorola. Both Android 12. Depending on the fill type you set, you get the top left angle, or the top 50% of the phone.
> 
> Does that happen with the latest NewPipe version (0.23.2) too?

Unfortunately it does. I use newpipe a few hours everyday with no exception, and have been experiencing this everyday since at least October 2021, and i update as soon as updates are available. Had a Galaxy s21+, now have a Motorola Edge 30 Pro. Both vanilla, as bought, no root, ecc.

**Comment by hammolo** [comment]
Just to make sure, I deleted all data and cache from the app, restarted the app, and the issue is still there. Did a short video to show it, since the video is captured on the phone itself, when rotating the phone in landscape the video doesn't fill up the screen, contrary to what I see on screen. Imagine it does, and about half the time it takes about half the width of the screen, instead of the whole width

https://user-images.githubusercontent.com/94056262/185481559-750cf96c-03a2-4fc2-ac02-9268694233b6.mp4

.

**Comment by ShareASmile** [comment]
@pedro1977uk is this still a issue on latest v0.24.1

**Comment by pedro1977uk** [comment]
I don't watch it as much as before however it did occur once the other day, but I believe it's a lot better than before.
 
 
  On Mon, 26 Dec 2022 at 16:44, ***@***.***> wrote:   


@pedro1977uk is this still a issue on latest v0.24.1
